spec/models/piece_spec.rb

```diff
@@ -1,4 +1,92 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
+  before :all do
+    Game.destroy_all
```

Comment:
I think you won't need `Game.destroy_all`. Rails/rspec clears the database for you already.